frontend/packages/component-library/src/cms/footer/Footer.tsx

```diff
@@ -0,0 +1,118 @@
+import classNames from 'classnames';
+import {
+  Key,
+  HTMLAttributes,
+  AnchorHTMLAttributes,
+  ImgHTMLAttributes,
+} from 'react';
+
+import FontAwesomeV6Icon, {FontAwesomeV6IconProps} from '@/fontAwesomeV6Icon';
+
+import moduleStyles from './footer.module.scss';
+
+export interface SiteLink extends AnchorHTMLAttributes<HTMLAnchorElement> {
+  key: Key;
+  label: string;
+  href: string;
+}
+
+export interface SocialLink extends AnchorHTMLAttributes<HTMLAnchorElement> {
+  key: Key;
+  label: string;
+  href: string;
+  icon: FontAwesomeV6IconProps;
+}
+
+export interface ImageLink extends AnchorHTMLAttributes<HTMLAnchorElement> {
+  key: Key;
+  label: string;
+  href: string;
+  image: ImgHTMLAttributes<HTMLImageElement>;
+}
+
+export interface FooterProps extends HTMLAttributes<HTMLElement> {
+  /** Footer links */
+  siteLinks: SiteLink[];
+  /** Footer social links */
+  socialLinks: SocialLink[];
+  /** Footer bottom image links */
+  imageLinks: ImageLink[];
+  /** Footer copyright notices */
+  copyright: string;
+  /** Footer class */
+  className?: string;
+}
+
+/**
+ * ## Production-ready Checklist:
+ *  * (✔) implementation of component approved by design team;
+ *  * (✔) has storybook, covered with stories and documentation;
+ *  * (✔) has tests: test every prop, every state and every interaction that's js related;
+ *  * (see ./__tests__/Footer.test.tsx)
+ *  * (✔) passes accessibility checks;
+ *
+ * ### Status: ```Ready for dev```
+ *
+ * Design System: Footer Component.
+ * Acts as the main page footer.
+ */
+const Footer: React.FC<FooterProps> = ({
+  siteLinks,
+  socialLinks,
+  imageLinks,
+  copyright,
+  className,
+  ...HTMLAttributes
+}) => (
+  <footer
+    {...HTMLAttributes}
+    className={classNames(moduleStyles.footer, className)}
+  >
+    <div className={moduleStyles.footerContent}>
+      <label
+        className={moduleStyles.footerLinkListToggle}
+        aria-label="Click to expand or collapse Site links"
```

Comment:
Should we include a prop here for translation support in the future, or is it ok to launch with just English?